contracts/dao/nopeDAO/GovernorQuickReaction.sol

```diff
@@ -0,0 +1,32 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity ^0.8.4;
+
+import {GovernorCountingSimple} from "@openzeppelin/contracts/governance/extensions/GovernorCountingSimple.sol";
+
+/**
+ * @dev Extension of {Governor} for quick reaction, whereby a proposal
+ *      is executed immediately once Quorum is reached.
+ */
+abstract contract GovernorQuickReaction is GovernorCountingSimple {
```

Comment:
Inherits from `GovernorCountingSimple` as need `_quorumReached` etc and some vote counting logic to be implemented. Would prefer to be inheriting from a general standard interface like `GovernorCounting` that is vote method agnostic

Reply:
This should just inherit from Governor which defines _quorumReachedAbstractly.

I have a slight preference for explicitly enumerating the state rather than using this inheritance supercall because I think that is cleaner and more auditable/readable.

Reply:
Yep great point on the inheritance from Governor, and then rely on any end contract implementing/inheriting from something that implements `_quorumReached`. Have updated